src/app/choose-us/choose-us.component.scss

```diff
@@ -0,0 +1,144 @@
+@import 'src/assets/styles/variables';
+
```

Comment:
there is extensive use of 
**display: flex;
 flex-direction: column;**
please find a way so that you don't have to repeat this same code.